### DISCUSSION TITLE
Here is info on how to install Bionic on a bare metal host

### DISCUSSION THREAD

**Discussion by tbendien** [bionic-gpt/bionic-gpt#223] [General]
Thanks to Ian and Dio for your extensive help in getting the bare metal runbook going!
I am starting this thread to share what we are learning, every day we make progress.

Here is my blog where I am updating the runbooks I am developing:
https://gtcyberlabs.org/f/install-bionic-gpt-on-an-ubuntu-bare-metal-host-with-a-nvidia-gpu

In short, there are several ways to install Bionic on a bare metal, in general, you need a minimum of a 8GB GPU for current 7B models, a modest CPU and 16GB RAM..

I am running Bionic on a NVIDIA RTX 4000 8GB Quadro. I am also running Bionic on a NVIDIA RTX A4000 16GB for 13B models with great success. 

We have also tested running Bionic on a workstation with 4 GPU's, (2 x 8GB RTX + 2 x 16GB RTX) which worked to load Mixtral 8x7B MOE, but it was not fast, due to the data traversing the PCIe bus (IMHO) vs running a model that is entirely loaded into VRAM on a single GPU. My workstation has an i3 CPU and 16GB RAM.

We also have a yaml that installs PGAdmin so you can connect to the audit trail database which is really useful. Thanks to Dio for creating SQL that allows you to see user chat history for audit purposes and analytics etc.

Keep an eye on my blog where I will update the runbook soon with a whole lot more info.

**Comment by ianpurton** [comment]
Thanks, looks great.
